### ISSUE TITLE
libtatsu-1.0 needed for app-pda/libimobiledevice-9999

### ISSUE THREAD

**Issue by kostadinsh** [xoores/gentoo-overlay#2]
Hey, could you package https://github.com/libimobiledevice/libtatsu, since it appears to be required for libimobiledevice now?

```
configure: error: Package requirements (libtatsu-1.0 >= 1.0.3) were not met:

Package 'libtatsu-1.0' not found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables libtatsu_CFLAGS
and libtatsu_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```

I can try and make a PR for it if you are unable to.

**Comment by xoores** [comment]
Thanks for the info, I'll try to get to it in a day or two :slightly_smiling_face:

**Comment by xoores** [comment]
Should be fixed in 84c109e13f5e3cee0a5ea2b9113d1df57e430e5c - closing :slightly_smiling_face: 

If you have any issues, let me know.
